### ISSUE TITLE
New Feature : support \alpha<tab> and \UNICODE NAME<tab> completion in Editor ?

### ISSUE THREAD

**Issue by Carreau** [spyder-ide/spyder#20896]
This is not a feature that I directly request but I was part of a discussion between 2 math educators yesterday that were comparing Jupyter Notebook and Spyder for teaching and I ma just here to report a difference and suggest a new feature.

IPython console (Qt and terminal, even the one present in Spyder) as well as Jupyter Notebook and lab support completion from `\<latex>` or `\<unicode name>` to the corresponding unicode symbol.
ie `\al<tab>` suggest `\aleph`, `\alpha` and `\alpha<tab>` -> `α`. Same with for example `\ROMAN NUMERAL EIGHT`. This is limited to valid Python identifiers. Note that there is also `\α<tab>` -> `\alpha` to make how to type a character discoverable. 

Would it be possible to also have this completion in the Spyder editor ?

You should be able to find all the latex->unicode and reverse mapping in IPython itself

https://github.com/ipython/ipython/blob/main/IPython/core/completer.py

**Comment by dalthviz** [comment]
Hi @Carreau , sorry for the late response and thank you for the feedback! Makes sense to me :+1: but just in case, what do you think @spyder-ide/core-developers ?

**Comment by Carreau** [comment]
No worries for the late reply. I you think this is possible, it may also be a good first issue.

As an addendum and in particular with respect with it working only with valid Python identifiers, we had the request to have it work for all unicode/latex in strings context, but never took the time to implement it.

**Comment by jitseniesen** [comment]
See also the very old issue #879 which has some related discussion, more or less arriving at the same point.

**Comment by ccordoba12** [comment]
Good point @jitseniesen! Perhaps we should close this issue in favor of that one?
